src/components/Counter/__stories__/counter.stories.mdx

```diff
@@ -11,14 +11,18 @@ import "./counter.stories.scss";
 
 export const metaSettings = createStoryMetaSettings({
   component: Counter,
-  enumPropNamesArray: ["size", "color", "kind", ]
+  enumPropNamesArray: ["size", "color", "kind"]
 });
 
 <Meta
   title="Feedback/Counter"
-  component={ Counter }
-  argTypes={ metaSettings.argTypes }
-  decorators={ metaSettings.decorators }
+  component={Counter}
+  argTypes={{
+    ...metaSettings.argTypes,
+    wrapperClassName: { table: { disable: true } },
+    "data-testid": { table: { disable: true } }
```

Comment:
For line 23, the typescript migration guide dictated migrated component props should extend `VibeComponentProps`. That ended up adding `data-testid` as a prop to the component. Storybook picks this up and adds it to the Props section in the docs, also adding a control for it. Since this is not in the existing docs this line is to make Storybook ignore this prop.
https://github.com/mondaycom/monday-ui-react-core/blob/c6519783644b8f21e6f44d6a121b989bd88de679/TYPESCRIPT_MIGRATION.md?plain=1#L11

Reply:
Got it, great 👍